features/performance/deploy.sh

```diff
@@ -11,9 +11,14 @@ if [ "${RTNODES}" == "" ] ; then
     exit 1
 fi
 
-
 # W/A for https://bugzilla.redhat.com/show_bug.cgi?id=1777150
-oc label machineconfigpool/worker worker=
+# Apply 'worker=""' label only if not set already.
+MCPWORKER=$(oc get mcp --selector=worker="" --no-headers)
+if [ "${MCPWORKER}" == "" ] ; then
```

Comment:
This could be just 
`if [ -z ${MCPWORKER} ] ; then`

Reply:
I find the empty string more readable so this is not an issue for me.

Reply:
ok then